lib/classroom/models/concerns/extensions.rb

```diff
@@ -45,3 +45,28 @@ def deep_compact
     end
   end
 end
+
+
+class String
+  def boolean_value
+    downcase.strip == 'true'
+  end
```

Comment:
Isn't there any boolean parse method around?

Reply:
I didn't found it

Reply:
Yeah, I didn't found one either